website_analytics_piwik/models/website.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why have you removed the encoding?

Reply:
Because utf-8 is default encoding for Python 3